x/crosschain/keeper/msg_server_vote_outbound_tx.go

```diff
@@ -211,6 +211,7 @@ func (k msgServer) VoteOnObservedOutboundTx(goCtx context.Context, msg *types.Ms
 	if err != nil {
 		// do not commit tmpCtx
 		cctx.CctxStatus.ChangeStatus(types.CctxStatus_Aborted, err.Error())
+		cctx.GetCurrentOutTxParam().TxFinalizationStatus = types.TxFinalizationStatus_Executed
```

Comment:
where does this `TxFinalizationStatus_Executed` enum get read (or checked against)? Isn't `CctxStatus_Aborted/CctxStatus_Reverted` enough to tell a cctx's status is finalized

Reply:
It's not used as of now, because Finalization and Execution are the same process, it will be used when we separate it out in the future